Documentation/ceph-upgrade.md

```diff
@@ -57,6 +57,11 @@ those releases.
 * Rook no longer deploys Prometheus rules from the operator. If you have been relying on Rook to
   deploy prometheus rules in the past, please follow the upgrade instructions [below](#prometheus).
 
+* Due to a number of Ceph issues and changes, Rook officially only supports Ceph
+  v16.2.7 or higher for CephNFS. If you are using an earlier version, we highly recommend you
```

Comment:
Since we don't support earlier versions, i'd say we just tell them to upgrade instead of saying it's recommended
```suggestion
  v16.2.7 or higher for CephNFS. If you are using an earlier version,
```